oauth2_http/java/com/google/auth/oauth2/SecureSessionAgent.java

```diff
@@ -188,17 +190,28 @@ private SecureSessionAgentConfig getSecureSessionAgentConfigFromMDS() {
 
     String plaintextS2AAddress = "";
     String mtlsS2AAddress = "";
+    Object s2aAddressConfig = responseData.get(S2A_JSON_KEY);
```

Comment:
nit: Can you case to Map<String, Object> here so it doesn't need to be done twice below?

Reply:
Done in the latest commit